onmt/modules/GRU.lua

```diff
@@ -28,7 +28,7 @@ Parameters:
   * `layers` - Number of layers
   * `inputSize` - Size of input layer
   * `hiddenSize` - Size of the hidden layers
-  * `dropout` - Dropout rate to use (should be in $$[0,1]$$ range.
+  * `dropout` - Dropout rate to use (in $$[0,1]$$ range, if negative, force dropout even on first layer).
```

Comment:
good idea - I added the option in 151a48a. the value is internal set when assembling complex encoders, but we can possibly use it for adding dropout between lookup and first LSTM